cmd/cri-containerd/options/options.go

```diff
@@ -118,29 +118,29 @@ func (c *CRIContainerdOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&c.ContainerdConfig.Runtime, "containerd-runtime",
 		"io.containerd.runtime.v1.linux", "The runtime used by containerd.")
 	fs.StringVar(&c.ContainerdConfig.RuntimeEngine, "containerd-runtime-engine",
-		"", "Runtime engine used by containerd. (default = \"\" uses containerd default)")
+		"", "Runtime engine used by containerd. Use containerd default if not specified.")
 	fs.StringVar(&c.ContainerdConfig.RuntimeRoot, "containerd-runtime-root",
-		"", "The directory used by containerd for runtime state. (default = \"\" uses containerd default)")
+		"", "The directory used by containerd for runtime state. Use containerd default if not specified.")
 	fs.BoolVar(&c.PrintVersion, "version",
 		false, "Print cri-containerd version information and quit.")
 	fs.StringVar(&c.NetworkPluginBinDir, "network-bin-dir",
 		"/opt/cni/bin", "The directory for putting network binaries.")
 	fs.StringVar(&c.NetworkPluginConfDir, "network-conf-dir",
 		"/etc/cni/net.d", "The directory for putting network plugin configuration files.")
 	fs.StringVar(&c.StreamServerAddress, "stream-addr",
-		"", "The ip address streaming server is listening on. Default host interface is used if this is empty.")
+		"", "The ip address streaming server is listening on. Use default host interface if not specified.")
 	fs.StringVar(&c.StreamServerPort, "stream-port",
 		"10010", "The port streaming server is listening on.")
 	fs.StringVar(&c.CgroupPath, "cgroup-path",
-		"", "The cgroup that cri-containerd is part of. By default cri-containerd is not placed in a cgroup.")
+		"", "The cgroup that cri-containerd is part of. Do not place cri-containerd in any cgroup if not specified.")
 	fs.BoolVar(&c.EnableSelinux, "enable-selinux",
-		false, "Enable selinux support. (default false)")
+		false, "Enable selinux support.")
```

Comment:
I'd tell them the default here.

Reply:
Golang boolean flag is working like this:
* `cri-containerd` will not enable selinux.
* `cri-containerd --enable-selinux` will enable it.

Given so I feel like this fine here. :)

Reply:
ah ok not a true false thing.. my bad